script.readme

```diff
@@ -0,0 +1,2 @@
+The script starts with running codes for cleaning a raw(bronze) customer and order data sets into an intermediate table(silver data) for further business analysis.
```

Comment:
@Iriakuma please reference the link to how use markup language to create well formatted readme files.

Reply:
Edited. Also I think I am not sure what is expected of me concerning the readme file and does it appear on the readme.md session of the main code when you accept the pull request?

Reply:
Once I approve the pr, and you merge it, then it will appear on the master/main branch.
The goal of the readme is generally to:
1. describe what the project does
2. provides instructions on how to run/generate the project

Try something rather like this structure:
## Overview
This application ingests data from an online store. It cleans the data and then generates valuable insights for xyz..

## Data Structure
The data follows x architecture and is separated in the following schemas:
- Bronze...
- Silver...